packages/signature-controller/src/SignatureController.ts

```diff
@@ -185,8 +193,14 @@ export class SignatureController extends BaseController<
    * @param options.messenger - The restricted controller messenger for the sign controller.
    * @param options.state - Initial state to set on this controller.
    * @param options.trace - Callback to generate trace information.
+   * @param options.decodingApiUrl - Api used to get decoded data for permits.
```

Comment:
Minor, but since we now type the option properties, do we need this constructor JSDoc also?

Reply:
Constructor JS docs are existing, would you suggest removing them all ?